src/components/DirectLinks/DirectLinks.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from './DirectLinks.module.scss';
+import { LINKS } from './DirectLinks.constants';
+import { ReactComponent as OutsideLink } from 'assets/icons/Outside-Link.svg';
+
+export default function DirectLinks() {
+  return (
+    <div className={styles.DirectLinks}>
+      <div className={styles.InnerContainer}>
```

Comment:
For what purpose did you add InnerContainer? The whole component is centered within `$boxed-width`right? Just add the InnerContainer styles to `.DirectLinks`

Reply:
Yep, removed it!